app/1.3.x/loadbalancing.md

```diff
@@ -189,39 +189,42 @@ to this target it will query the nameserver again.
 
 ### Balancing algorithms
 
-By default a ring-balancer will use a weighted-round-robin scheme. The alternative
-would be to use the hash-based algorithm. The input for the hash can be either
-`none`, `consumer`, `ip`, `header`, or `cookie`. When set to `none` the
-weighted-round-robin scheme will be used, and hashing will be disabled.
-
-There are two options, a primary and a fallback in case the primary fails
-(e.g., if the primary is set to `consumer`, but no consumer is authenticated)
-
-The different hashing options:
-
-- `none`: Do not use hashing, but use weighted-round-robin instead (default).
-
-- `consumer`: Use the consumer id as the hash input. This option will fallback
-  on the credential id if no consumer id is available (in case of external auth
-  like ldap).
-
-- `ip`: The remote (originating) IP address will be used as input. Review the
-  configuration settings for [determining the real IP][real-ip-config] when
-  using this.
-
-- `header`: Use a specified header (in either `hash_on_header` or `hash_fallback_header`
-  field) as input for the hash.
-
-- `cookie`: Use a specified cookie name (in the `hash_on_cookie` field) with a
-  specified path (in the `hash_on_cookie_path` field, default `"/"`) as input for
-  the hash. If the cookie is not present in the request, it will be set by the
-  response. Hence, the `hash_fallback` setting is invalid if `cookie` is the primary
-  hashing mechanism.
-
-The hashing algorithm is based on 'consistent-hashing' (or the 'ketama principle')
-which makes sure that when the balancer gets modified by changing the targets
-(adding, removing, failing, or changing weights) only the minimum number of
-hashing losses occur. This will maximize upstream cache hits.
+The ring-balancer supports three load-balancing algorithms: `round-robin`,
+`consistent-hashing`, and `least-connections`. By default, a ring-balancer
+will use the `round-robin` algorithm.
+
+When using the `hash-based` algorithm, the input for the hash can be either
+`none`, `consumer`, `ip`, `header`, or `cookie`. When set to `none`, the
+`round-robin` scheme will be used, and hashing will be disabled. The `hash-based`
+algorithm supports a primary and a fallback hashing attribute; in case the primary
+fails (e.g., if the primary is set to `consumer`, but no Consumer is authenticated),
+the fallback attribute is used.
+
+Supported hashing attributes are:
+
+- `none`: Do not use `hash-based`, but use `round-robin instead` (default).
+- `consumer`: Use the Consumer ID as the hash input. This option will fallback
+  on the Credential ID if no Consumer ID is available (in case of an external
+  authentication mechanism like LDAP).
```

Comment:
```suggestion
- `consumer`: Use the Consumer ID as the hash input. If no Consumer ID is available, it will fall back on the Credential ID (for example, in case of an external authentication mechanism like LDAP).
```